TodoList/View Controllers/TodoListViewController.swift

```diff
@@ -27,10 +33,7 @@ class TodoListViewController: UITableViewController {
 
         tableView.register(cellType: ItemTableViewCell.self)
         tableView.tableFooterView = footer
-        tableView.estimatedRowHeight = 44
-        tableView.rowHeight = UITableViewAutomaticDimension
-        tableView.estimatedSectionFooterHeight = 80
-        tableView.sectionFooterHeight = UITableViewAutomaticDimension
```

Comment:
How did you replace `UITableViewAutomaticDimension`? 
I have looked through the changes and could not find it.

Reply:
`UITableViewAutomaticDimension` is set by default when done programmatically.

https://developer.apple.com/documentation/uikit/uitableview/1614852-rowheight
> The default value of rowHeight is automaticDimension. Note that if you create a self-sizing cell in Interface Builder, the default row height is changed to the value set in Interface Builder